libs/loggers/video_logger.py

```diff
@@ -0,0 +1,177 @@
+import cv2 as cv
+import numpy as np
+import os
+import shutil
+
+from libs.utils import visualization_utils
+from tools.environment_score import mx_environment_scoring_consider_crowd
+
+
+class VideoLogger:
+
+    def __init__(self, config, source: str, logger: str, live_feed_enabled: bool):
+        self.config = config
+        self.camera_id = self.config.get_section_dict(source)["Id"]
+        self.resolution = tuple([int(i) for i in self.config.get_section_dict("App")["Resolution"].split(",")])
+        self.birds_eye_resolution = (200, 300)
+        self.out = None
+        self.out_birdseye = None
+        self.live_feed_enabled = live_feed_enabled
+        self.track_hist = dict()
+
+    def start_logging(self, fps):
+        if not self.live_feed_enabled:
+            return
+        self.out, self.out_birdseye = (
```

Comment:
Birdseye is not used at all in the dashboard. Should we remove it?

Reply:
Is not used now. However, I am not sure that we don't want to include it in the future.